annotator-core/src/main/java/edu/ucr/cs/riple/core/evaluators/AbstractEvaluator.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 Nima Karimipour
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package edu.ucr.cs.riple.core.evaluators;
+
+import com.google.common.collect.ImmutableSet;
+import edu.ucr.cs.riple.core.Config;
+import edu.ucr.cs.riple.core.Report;
+import edu.ucr.cs.riple.core.evaluators.graphprocessor.ConflictGraphProcessor;
+import edu.ucr.cs.riple.core.evaluators.suppliers.Supplier;
+import edu.ucr.cs.riple.core.metadata.graph.ConflictGraph;
+import edu.ucr.cs.riple.core.metadata.index.Fix;
+
+/** Abstract class for evaluators. */
+public abstract class AbstractEvaluator implements Evaluator {
+
+  /** Annotator config. */
+  protected final Config config;
+  /** Conflict graph to storing unprocessed fixes. */
+  protected final ConflictGraph graph;
+  /** Depth of analysis. */
+  protected final int depth;
+  /** Graph processor to process the graph. */
+  protected ConflictGraphProcessor analyzer;
```

Comment:
Why is this field named `analyzer` and not e.g. `processor` or even `graphProcessor`? Wouldn't that be clearer?

Reply:
That should have been `processor`. Thank you for catching this. c92be6aa18314d9eabca233a27bd3d1bb70ee5ae